packages/in_app_purchase/lib/src/billing_client_wrappers/sku_details_wrapper.dart

```diff
@@ -124,6 +125,16 @@ class SkuDetailsWrapper {
         type.hashCode,
         isRewarded.hashCode);
   }
+
+  /// Method to convert to the wrapper to the consolidated [ProductDetails] class.
```

Comment:
nit: I think it's worth warning that this conversion is one-way since we're losing data.